app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt

```diff
@@ -0,0 +1,145 @@
+package ch.sdp.vibester.activity
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Environment
+import android.util.Log
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup.LayoutParams
+import android.view.Window
+import android.widget.Button
+import android.widget.LinearLayout
+import android.widget.LinearLayout.LayoutParams.MATCH_PARENT
+import android.view.ViewGroup.LayoutParams.MATCH_PARENT
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.widget.TextView
+import android.widget.Toast
+import ch.sdp.vibester.R
+import org.w3c.dom.Text
+import java.io.*
+
+class DeleteSongsActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        requestWindowFeature(Window.FEATURE_NO_TITLE)
+        supportActionBar?.hide()
+
+        setContentView(R.layout.activity_delete_songs)
+
+        val layout: LinearLayout = findViewById(R.id.delete_songs_linear)
+        generateButtons(layout)
+    }
+
+    /*
```

Comment:
```suggestion
    /**
```
for comment this is one *, for doc it is ** : https://www.oracle.com/ch-fr/technical-resources/articles/java/javadoc-tool.html